orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1413,6 +1413,40 @@ void RegisterTrainingOpSchemas() {
         propagateShapeAndTypeFromFirstInput(ctx);
       });
 
+ONNX_CONTRIB_OPERATOR_SCHEMA(InPlaceAccumulatorV2)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc("In-place accumulator for tensors. Differs from older op by adding cotrol input for reset, and optional output buffer.")
```

Comment:
"cotrol input for reset" - is this referring to `overwrite_flag`? or `updated_flag`, which is an added output?

Reply:
Can you update the description to add this info? Something like: 
Differs from older op by adding "overwrite_flag" as an input and making output buffer as optional. Set overwrite_flag to false for gradient accumulation and to True for overwriting the accumulation buffer (equivalent to reset + train step)